yesod-test/test/main.hs

```diff
@@ -169,6 +170,15 @@ main = hspec $ do
                     addToken_ "body"
                 statusIs 200
                 bodyEquals "12345"
+            yit "can follow a link via clickOn" $ do
+              get ("/htmlWithLink" :: Text)
+              clickOn "a#thelink"
+              statusIs 200
```

Comment:
Should this test that the actual new page is returned after the link is clicked?